src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/FixedSchema.cs

```diff
@@ -441,15 +441,14 @@ public override string[] ExtractUriFromAttr(string attrName, string attrValue)
                 attrName.Equals("Fill", StringComparison.Ordinal) ||
                 attrName.Equals("Stroke", StringComparison.Ordinal))
             {
-                attrValue = attrValue.Trim();
-                if (attrValue.StartsWith(_contextColor, StringComparison.Ordinal))
+                ReadOnlySpan<char> attrValueSpan = attrValue.AsSpan().Trim();
+                if (attrValueSpan.StartsWith(_contextColor, StringComparison.Ordinal))
                 {
-                    attrValue = attrValue.Substring(_contextColor.Length);
-                    attrValue = attrValue.Trim();
-                    string[] tokens = attrValue.Split(new char[] { ' ' });
-                    if (tokens.GetLength(0) >= 1)
+                    attrValueSpan = attrValueSpan.Slice(_contextColor.Length).Trim();
+                    int spacePos = attrValueSpan.IndexOf(' ');
+                    if (spacePos >= 0)
                     {
-                        return new string[] { tokens[0] };
+                        return new string[] { attrValueSpan.Slice(0, spacePos).ToString() };
```

Comment:
Seems it changed the behavior

Reply:
How so?

Reply:
@stephentoub Thank you. I wrote some test code, and found that the behavior has not changed.